src/settings/index.ts

```diff
@@ -552,6 +552,8 @@ export const DEFAULT_AVATARS = [
   "/avatars/default-profile-pic-2.png",
   "/avatars/default-profile-pic-3.png",
   "/avatars/default-profile-pic-4.png",
+  "/avatars/default-profile-pic-5.png",
```

Comment:
These are old files these urls should be changed and the files should be moved to `src/assets`

Reply:
@denisdimitrov I don't understand what's wrong and why they are old? I've just changed avatars

and why should they be in src/assets and not in src/assets/avatars? they were there before and the other avatars are there too

Reply:
@dmitrii-zolotukhin Look at the files of the PR, they are currently in the `public` folder, which is a bad practice and React will 'flash' when loading the files.

Reply:
They are always going to be bundled even when not being used, if you add them to `src/assets` they will be used and imported only in the correct file, because of our lazy loading.

Reply:
Right now you can use `/avatars/default-profile-pic-5.png` because they are in public, if you change them to `src` you will have to import them, you can see `settings.ts` and the usage there.

Reply:
fixed in 8e852da134b5d0818d043ba28933080e00e3094c